domain-server/resources/web/ssl-acme-client/index.html

```diff
@@ -0,0 +1,117 @@
+            </select>
+            <a href="" id="terms-of-service">Terms of Service</a>
+            <input id="directory-url" type="text"/>
+        </div>
+
+        <div id="auth-container">
+            <label>Authentication method</label>
+            <select id="auth-select">
+                <option value="account-key-only">Account Key</option>
+                <option class="eab-option zero-ssl-auth-option zero-ssl-auth-option-default" value="zero-ssl-email">ZeroSLL Email</option>
+                <option class="eab-option zero-ssl-auth-option" value="zero-ssl-api-key">ZeroSSL API Key</option>
+                <option class="eab-option" value="id-mac">ID and MAC key</option>
+            </select>
+            <input id="zero-ssl-auth-input" placeholder="ZeroSSL Email" type="text" />
+            <input id="eab-kid-input" placeholder="KID" type="text" />
+            <input id="eab-mac-input" placeholder="MAC" type="text" />
+        </div>
+
+        <div id="other-settings-container">
+            <label>Challenge completion method</label>
+            <select id="challenge-select">
+                <option value="server">HTTP Server</option>
+                <option value="files">Files</option>
+                <option value="manual">Manual</option>
+            </select>
+            <br />
+            <label>Account Key</label>
+            <input id="account-key-path" type="text" placeholder="<Application Data Path>/acme_account_key.pem" />
+            <input id="account-key-upload" type="file" />
+            <br />
+            <input id="account-key-reset" type="button" value="Reset Account Key" />
+        </div>
+        <br />
+
+        <div id="domains-container">
+            <label>Domains</label>
+            <div id="domain-inputs">
+                <div class="domain-input-container">
+                    <input type="text" class="domain-name-input" placeholder="Domain Name" />
+                    <input type="text" class="domain-dir-input" placeholder="Domain Directory" />
+                    <input type="button" class="remove-domain-button"  value="Remove" />
+                </div>
+            </div>
+            <input id="add-domain-button" type="button" value="Add Domain" />
+        </div>
+
+        </div>
+
+        <h4>Certificate Paths</h4>
+
+        <div id="paths-container">
+            <label>Certificate Directory</label>
+            <input id="cert-dir" type="text" placeholder="Application Data Path" />
+            <br />
+            <label>Certificate</label>
+            <input id="cert-name" type="text"/>
+            <input id="cert-upload" type="file" />
+            <br />
+            <label>Key</label>
+            <input id="cert-key-name" type="text" />
+            <input id="cert-key-upload" type="file" />
+            <!-- <br /> -->
+            <!-- <label>Expiry date<label> -->
+            <!-- <label>Renewal date<label> -->
```

Comment:
Remove commented-out code?

Reply:
Yep, left it there to remember, but there is a lot more work needed on the UI than just that, so not relevant anymore.